x/stakeibc/keeper/icacallbacks_detokenize.go

```diff
@@ -25,8 +25,19 @@ func DetokenizeCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, a
 		return errorsmod.Wrapf(types.ErrUnmarshalFailure, "unable to unmarshal detokenize callback: %s", err.Error())
 	}
 	chainId := detokenizeCallback.Deposit.ChainId
+	deposit := detokenizeCallback.Deposit
```

Comment:
Sorry for stupid question but it is on the detokenize not on the transfer complete callback which the amount we could query of the native token in delegation account would change?

Reply:
yeah transfer only updates the balance, detokenization updates the delegation